beacon-chain/core/feed/state/events.go

```diff
@@ -55,11 +59,3 @@ type InitializedData struct {
 	// GenesisValidatorsRoot represents state.validators.HashTreeRoot().
 	GenesisValidatorsRoot []byte
 }
-
-// ReorgData is the data alongside a reorg event.
-type ReorgData struct {
-	// NewSlot is the slot of new state after the reorg.
-	NewSlot types.Slot
-	// OldSlot is the slot of the head state before the reorg.
-	OldSlot types.Slot
-}
```

Comment:
This is failing builds.

beacon-chain/rpc/validator/assignments_test.go:348:11: undefined: "github.com/prysmaticlabs/prysm/beacon-chain/core/feed/state".ReorgData
beacon-chain/rpc/validator/assignments_test.go:356:11: undefined: "github.com/prysmaticlabs/prysm/beacon-chain/core/feed/state".ReorgData